charts/external-dns/templates/deployment.yaml

```diff
@@ -157,10 +157,10 @@ spec:
             {{- toYaml . | nindent 12 }}
           {{- end }}
           ports:
-            - name: http-webhook
+            - name: http-wh-private
               protocol: TCP
               containerPort: 8888
-            - name: http-wh-metrics
+            - name: http-webhook
```

Comment:
Wouldn't `http-wh` be more consistent?

Reply:
@kimsondrup I'm not sure why a localhost bound port is exposed here? See my comment above.

Reply:
@stevehipwell only named the "private" port to make it more explicit that it is a thing.
Just removed `http-wh-private` since it does not serve any purpose